include/libtorrent/alert_types.hpp

```diff
@@ -1810,8 +1810,6 @@ namespace libtorrent {
 
 		static constexpr alert_category_t static_category = alert::status_notification;
 		std::string message() const override;
-
-		error_code const error;
```

Comment:
I think removing this would alter the API compared to 1.1, wouldn't it?
If so, I think it should be deprecated instead.

Reply:
right, will do that